unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarNamesHUD/AvatarNamesHUDController.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.Generic;
+using DCL;
+
+namespace AvatarNamesHUD
+{
+    public class AvatarNamesHUDController : IHUD
+    {
+        private int maxAvatarNames;
+
+        private BaseDictionary<string, PlayerStatus> otherPlayersStatus => DataStore.i.player.otherPlayersStatus;
+
+        internal IAvatarNamesHUDView view;
+        internal readonly HashSet<string> trackingPlayers = new HashSet<string>();
+        internal readonly LinkedList<string> reservePlayers = new LinkedList<string>();
+
+        internal virtual IAvatarNamesHUDView CreateView() { return AvatarNamesHUDView.CreateView(); }
+
+        public AvatarNamesHUDController() : this(200) { }
```

Comment:
Can we move this 200 to a constant? so we have more context about what it is

Reply:
Done!